model/marshal_test.go

```diff
@@ -400,6 +400,20 @@ func TestMarshalURL(t *testing.T) {
 	assert.Equal(t, in, out)
 }
 
+func TestMarshalURLPathEmpty(t *testing.T) {
+	var empty fastjson.Writer
+	(&model.URL{
+		Hostname: "example.com",
+		Path:     "",
+		Protocol: "http",
+	}).MarshalFastJSON(&empty)
```

Comment:
```suggestion
	in := model.URL{
		Hostname: "example.com",
		Path:     "",
		Protocol: "http",
	}

	var w fastjson.Writer
	w.MarshalFastJSON(&w)
```

For consistency with other tests.